src/js/gameframe.js

```diff
@@ -466,8 +466,7 @@ var GameFrame;
             // capture joy stick
             if (!joystick
                     && Object.values(special).indexOf(key) + 1
-                    && typeof nipplejs !== 'undefined'
-                    && window.matchMedia('(display-mode: standalone)').matches) {
```

Comment:
I think this makes it show up on desktop too though. Which is not super desirable. This is live on the API right? Try clicking on https://api.carolinaignites.org/app/pggwqZMxZPzEJr1z/

Maybe it's fine